linux/keyman-config/keyman_config/get_kmp.py

```diff
@@ -44,6 +43,15 @@ def get_download_folder():
 	"""
 	Folder where downloaded files will be saved.
 
+	Returns:
+	    str: path of user keyman cache folder
+	"""
+	return keyman_cache_dir()
+
+def keyman_cache_dir():
+	"""
+	User keyman cache folder
+
 	Returns:
 	    str: path of user keyman cache folder
```

Comment:
Could also document the cache folder will be created if it doesn't exist

Reply:
good point, done in next commit